src/instance.rs

```diff
@@ -21,17 +21,22 @@ use crate::{
 };
 
 #[derive(Debug)]
-pub struct Instance {
+pub struct Instance<'a> {
     instance: wasm_module_inst_t,
+    _phantom: PhantomData<&'a ()>,
```

Comment:
because `pub type wasm_module_inst_t = *mut WASMModuleInstanceCommon;`, I guess it should be something like:
```suggestion
    _phantom: PhantomData<WASMModuleInstanceCommon>,
```